extensions/yarn/README.md

```diff
@@ -42,6 +46,39 @@ In cases where Phylum still needs to process some of the packages, the command
 will exit with a warning **without** installing the packages. Once the analysis
 is complete, another attempt can be made.
 
-[phylum]: https://phylum.io
-[phylum-cli]: https://github.com/phylum-dev/cli
-[yarn]: https://yarnpkg.com/
+## Troubleshooting
+
+There are some common reasons this extension may fail.
+
+### Sandbox violations
+
+The underlying package manager may fail due to operating within the Phylum
+sandbox. These errors are usually displayed in the output and say something
+about not being able to access a particular file or network resource.
+
+One possibility is that this is a valid sandbox violation that has not been
+accounted for with an exception. If you believe this is the case, please
+[submit an issue][issue] or [contact us][contact] to discuss adding exceptions.
+
+Another possibility is that a malicious action was taken by one of the packages.
```

Comment:
Malicious activity would only be a failure if it fails loudly, which is probably the minority of cases.

Reply:
Blocking malicious activity is why users want to use these extensions (based on feedback from the user on Discord that initiated this PR). Noting it is worthwhile, regardless of the frequency of occurrence. Alternate wording suggestions are welcome.

Reply:
I'm not saying malicious activity isn't the only cause of failure. I'm saying malicious activity might be blocked without causing failure.